.github/workflows/pull-request-sonar.yml

```diff
@@ -28,8 +28,8 @@ jobs:
         run: yarn format
       - name: Run build
         run: yarn build
-      - name: Run test and write coverage
-        run: yarn test:coverage
```

Comment:
The checks.yml is still using the `test:coverage` command, if that command doesn't work for Jest do you think we should change that one too?

Reply:
I did this in a PR that has now been merged so a rebase should sort it 👍